service/src/main/java/org/sunbird/actor/user/UserUpdateActor.java

```diff
@@ -153,6 +161,13 @@ private void updateUser(Request actorMessage) {
     // Check if the user is Custodian Org user
     boolean isCustodianOrgUser = isCustodianOrgUser((String) userDbRecord.get(JsonKey.ROOT_ORG_ID));
     encryptExternalDetails(userMap, userDbRecord);
+    try {
+      Map map = ProfileUtil.toMap(userMap.get(JsonKey.PROFILE_DETAILS).toString());
```

Comment:
this will throw NPE, its better to null check here before calling toString method

Reply:
updated the code properly